tests/framework/shim/windows_shim.cpp

```diff
@@ -35,6 +35,10 @@
 
 #include "detours.h"
 
+#define LOADER_PLATFORM_SKIP_FUNCTION_DEFINITIONS
+#include "loader/vk_loader_platform.h"
+#undef LOADER_PLATFORM_SKIP_FUNCTION_DEFINITIONS
```

Comment:
I'm not sure about this change - I presume you are doing this to be able to re-use the reg-key definitions?

From the perspective of having the test be able to detect unexpected registry key changes, it would probably be better to just repeat the logic to build up the correct Vulkan/VulkanSC regkey in the test file itself. 

@aqnuep not sure what you think from the maintainability perspective.

Reply:
Yes, I agree that this way we lose coverage for accidental changes.